### PR TITLE
Important documentation change to avoid mistakes.

### PR DESCRIPTION
**Pull request by TwistedLogic**

### CODE REVIEW

**Comment by bschrammIBM** [comment]
Thank you for this edit. Please give us a few days, so that the engineering team can review it.

**Review by raymondfeng — APPROVED** [review]

File: pages/en/lb3/Redis-connector.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Edit `datasources.json` to add other properties that enable you to connect the
     <tr>
       <td>host</td>
       <td>String</td>
-      <td>Database host name</td>
+      <td>Database host name. When this property is set, also the property **port** needs to be set.</td>
```
Comment (raymondfeng) [inline pages/en/lb3/Redis-connector.md]: We call `redis.createClient(port, host, options)`. See https://github.com/NodeRedis/node_redis#rediscreateclient. I think this line can be made more robust - https://github.com/strongloop-community/loopback-connector-redis/blob/master/lib/redis.js#L37. 

@TwistedLogic Do you want to fix the code?

Comment (TwistedLogic) [reply pages/en/lb3/Redis-connector.md]: I already submitted a PR.

Comment (raymondfeng) [reply pages/en/lb3/Redis-connector.md]: Got it - https://github.com/strongloop-community/loopback-connector-redis/pull/57

Comment (kjdelisle) [reply pages/en/lb3/Redis-connector.md]: I'd rephrase this to say `When this property is set, the port property **must also** be set.` or something to that effect.

**Review by kjdelisle — CHANGES_REQUESTED** [review]
Grammar nitpick. Once it's fixed, LGTM

**Review by bschrammIBM — APPROVED** [review]

**Comment by bschrammIBM** [comment]
merging as kjdelisle's suggested edit was completed.
Thank you for this contribution!

**Review by kjdelisle — APPROVED** [review]
